jetty-io/src/main/java/org/eclipse/jetty/io/SelectorManager.java

```diff
@@ -147,14 +147,13 @@ protected void execute(Runnable task)
     @ManagedAttribute(value = "Total number of keys in all selectors", readonly = true)
     public int getTotalKeys()
     {
-        if (_selectors == null || _selectors.length == 0)
-        {
-            return 0;
-        }
         int keys = 0;
-        for (final ManagedSelector selector : _selectors)
+        for (ManagedSelector selector : _selectors)
         {
-            keys += selector.getTotalKeys();
+            if (selector != null)
+            {
```

Comment:
Please drop the braces for single line statements.

Also, there is an extra new line at the end of this method, please remove that too.

Reply:
Done